lib/stream.c

```diff
@@ -1097,22 +1097,41 @@ int stream_put_prefix(struct stream *s, const struct prefix *p)
 }
 
 /* Put NLRI with label */
-int stream_put_labeled_prefix(struct stream *s, const struct prefix *p,
-			      mpls_label_t *label, bool addpath_capable,
-			      uint32_t addpath_tx_id)
+int stream_put_labeled_prefix(struct stream *s, const struct prefix *p, mpls_label_t *label,
```

Comment:
why is this in lib/ - it sure looks as if it's a bgp-protocol function?

Reply:
the diff here is still ... large? it'd be clearer just to get the bgp parts in, and then look at the lib/stream part.